ubi-ruby-fips/Description.md

```diff
@@ -0,0 +1,9 @@
+# ubi-ruby-fips
+ `ubi-ruby-fips` combines a [base UBI image](https://catalog.redhat.com/software/containers/ubi8/ubi/5c359854d70cc534b3a3784e) 
+ with Ruby compiled against the FIPS 140-2 compliant [OpenSSL module](https://www.openssl.org/docs/fips.html).  
+This image includes the following packages:
+
+* Ruby version `2.5.8`: compiled against the FIPS 140-2 compliant OpenSSL module.
+* Bundler version `2.1.4`.
+ 
```

Comment:
do you not want to mention the OpenSSL and PG versions as the other images do?
```
* OpenSSL version `1.0.2u`: built with  FIPS 140-2 compliant OpenSSL module version `2.0.16`.
* Postgres client version `10-10.14`: compiled against the FIPS 140-2 compliant OpenSSL module.
```

Reply:
The UBI doesn't have a specific version of openssl and Postgres client because we don't compile a dedicated version of it for UBI.
Because openssl on UBI 8 is FIPS out of the box it will be updated with the latest version they are pushing the the ubi 8 base os yum repo.